tests/test_stack.py

```diff
@@ -81,6 +81,35 @@ def test_stack_pointselection_some(self, synthetic_dataset):
             ["complex", "amplitude", "phase"]
         )
 
+    def test_stack_pointselection_nan(self):
+        data = np.random.rand(10, 10, 10) + 1j * np.random.rand(10, 10, 10)
+        data[0, 0, 0] = np.nan  # introduce a nan
```

Comment:
Here it is assumed that there is a "`nan`" value in the dataset, what if this point is on land and not water? the reason why this value is `nan` can be just a gap in the time series. Instead, I would suggest changing the behavior of the argument `threshold` from a single value to a tuple (threshold_min, threshold_max) to fix issue #44. So, the implementation would be something like 
```python
amplitude_dispersion = self._amp_disp()
mask = (amplitude_dispersion >= threshold_min) & (amplitude_dispersion <= threshold_max)
```

Reply:
As discussed, we keep the original masking with only the upper threshold.